app/views/content/non-uk-teachers/train-to-teach-in-england-as-an-international-student.md

```diff
@@ -70,7 +70,14 @@ keywords:
 
 $new-international-funding$
 
-Citizens of every country in the world can apply to train to teach in England.
+Citizens of every country in the world can apply to train to teach in England. When you plan your training, bear in mind that:
+
+- international candidates can expect to pay between £9,250 and £32,000, plus
+living costs, to train (depending on your immigration status and the subject you train to teach, you may get some financial support)
```

Comment:
@laurahermionetennant92 this is quite a long sentence. Could you remove the brackets and have it as 2 separate sentences? 

**the subject you train to teach**, does this need in at the end? the subject you train to teach in

Reply:
I've got some updates to the way we talk about fees on the other PR – will fix there